api/eid_service/views.py

```diff
@@ -101,7 +101,7 @@ def refresh(request):
             return HttpResponseBadRequest()
         authenticationRequest.restrictedId = get_result(authenticationRequest.sessionId)
         authenticationRequest.save()
-        return HttpResponseRedirect('https://' + request.get_host() + '/api/openid/login?eid_access_token=' + str(authenticationRequest.accessToken) + "&" + authenticationRequest.openidParameter)
+        return HttpResponseRedirect('https://' + request.get_host() + '/api/eidopenid/login?eid_access_token=' + str(authenticationRequest.accessToken) + "&" + authenticationRequest.openidParameter)
```

Comment:
preferably use django's [built-in method to redirect to other views](https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#django.urls.reverse)